gulpfile.js

```diff
@@ -58,18 +58,18 @@ function lesslint() {
 }
 
 function less() {
-	return gulp.src('src/**/*.less')
+	return gulp.src('src/sass/**/*.scss')
```

Comment:
Let's set the glob to `'src/**/*.scss'` as we could potentially have SASS files under a specific component in `/app`. Same apply to other globs in the PR.